topi/tests/python/test_topi_transform.py

```diff
@@ -185,6 +185,56 @@ def check_device(device):
         check_device(device)
 
 
+def verify_take(src_shape, indices_shape, indices_val=0, axis=0):
```

Comment:
add a testcase that contains several steps,  e.g. take+relu, there could be a possible bug here, schedule_injective should be able to solve take + relu if take is marked as injective, currently it is marked as ""